### DISCUSSION TITLE
FrappeBooks and Telegram

### DISCUSSION THREAD

**Discussion by GTGeek88** [frappe/books#800] [General]
Was looking around here and it looks interesting. But it does make me stop and wonder when I see the Telegram usage. That's a questionable app for a lot of people. Why associate the product with an app used by hate groups? Not that other platforms aren't misused, but Telegram has been the target of a lot of criticism. Why not have all communications here on GitHub?

**Comment by Isaac-GC** [comment]
Hi @GTGeek88!

Thanks for the question, I can understand where you are coming from in regards to Telegram!

The usage of Telegram is used for quicker support of issues and questions. While telegram in general *does* contains hate groups, they are only a very small subset of the overall user base, and telegram is still used by a lot of legitimate groups/people. Using telegram also makes it easier for users who aren't as technical or have Github accounts to have their questions asked/answered by the Frappe Books community.

If you aren't comfortable using Telegram, you're more than welcome to ask your questions as a Github Issue or Discussion! We support any/al questions in both telegram and here  :)

**Reply by GTGeek88** [reply]
I'm not sure creating a Telegram account is going to be less technical than creating a GitHub account, but perhaps so. I've avoided Telegram. Seems lots of scams come from there, as well as hate groups using it. It seems to have a "reputation." Anyway, thanks for your response.

**Comment by Isaac-GC** [comment]
@GTGeek88 I'm reopening this discussion as I think there might need to be an alternative to Telegram. It works, but when trying to help people... it'd be nice to break out into rooms/channels, or even just have different subjects for each room/channel (i.e updates, general help, developing frappe books help, etc)

My thoughts are to consider moving to an alternative platform such as Slack or Discord (preferably the former), but not sure of all the alternatives out there. Are there any like that, that you personally (or anyone else who's seeing this) has used and liked?

**Comment by GTGeek88** [comment]
Well, I'm not a fan of Discord. Don't know much about Slack, but, of course, it does seem pretty popular.

**Comment by stxm** [comment]
@Isaac-GC what about matrix? 
It Is arguably less popular than telegram/slacks but it is popular with the FOSS community
